tests/containers/containerd_rmt.pm

```diff
@@ -0,0 +1,63 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Test deploy a helm chart in a k3s
+# - install k3s, kubectl and helm
+# - test helm repo add, update, search and show all
+# - add bitnami repo
+# - test helm install with apache helm chart
+# - test helm list
+# - check the correct deployment of the helm chart
+# - cleanup system (helm and k3s)
+#
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use utils;
+use version_utils qw(is_sle);
+use serial_terminal 'select_serial_terminal';
+use registration qw(add_suseconnect_product get_addon_fullname);
+use containers::k8s;
+
+sub run {
+    my ($self) = @_;
+
+    select_serial_terminal;
+    systemctl 'stop firewalld';
+    install_k3s();
+    install_kubectl();
+    install_helm();
+
+    my $rmt_helm = get_var('RMTTEST_REPO', 'https://gitlab.suse.de/brunoleon/rmt-helm/-/archive/main/rmt-helm-main.zip');
+    # pull in the testsuite
+    assert_script_run("wget --quiet --no-check-certificate " . $rmt_helm);
+    assert_script_run("unzip rmt-helm-main.zip");
+    my $myvalue = get_var('MY_VALUE', 'https://gitlab.suse.de/QA-APAC-I/testing/-/raw/master/data/rmtcontainer/myvalue.yaml');
+    assert_script_run("wget --no-check-certificate " . $myvalue);
+    record_info('chart', script_output('cat myvalue.yaml'));
+    assert_script_run("helm install rmt ./rmt-helm-main -f myvalue.yaml");
```

Comment:
I think is better to use the flag -wait

Reply:
Also the name of the helm deployment should be randomized to prevent concurrent executions or previous leftovers (e.g. in case helm delete didn't work) or the execution was interrupted

Reply:
Forget my comment, you are using local k3s installation. This should be mandatory on a k8s in a cloud provider